torch/csrc/distributed/rpc/tensorpipe_utils.h

```diff
@@ -14,6 +14,13 @@
 
 namespace tensorpipe {
 class Message;
+#ifdef TP_NEW_API
+class Allocation;
+class Descriptor;
+#else
+using Allocation = Message;
+using Descriptor = Message;
+#endif // TP_NEW_API
```

Comment:
I assume there will be an immediate PR to remove these aliases?

Reply:
Right, this preprocessor switch is introduced to allow a clean update in fbcode and GitHub at the same time. The `TP_NEW_API` switch will never actually be enabled in GitHub, and the `#ifdef` will simply disappear in the next PR.